addons/test_mail/tests/test_message_track.py

```diff
@@ -84,30 +86,40 @@ def test_message_track_subtype(self):
 
     def test_message_track_template(self):
         """ Update some tracked fields linked to some template -> message with onchange """
-        self.record.write({'mail_template': self.env.ref('test_mail.mail_test_ticket_tracking_tpl').id})
-        self.assertEqual(self.record.message_ids, self.env['mail.message'])
-
-        with self.mock_mail_gateway():
-            self.record.write({
-                'name': 'Test2',
-                'customer_id': self.user_admin.partner_id.id,
+        sender_email = 'sender@test.lan'
```

Comment:
Why are we changing the test here actually ?

Reply:
:thinking: hu... why indeed..

I guess it might have been useful when I added that new notification_type but I really should just have added a test either way. Doesn't seem to add anything as it stands so I'll just remove it (and the test_model I'd borrowed from you)